server/protocols/webadmin-cli/src/main/java/org/apache/james/httpclient/JwtToken.java

```diff
@@ -19,15 +19,17 @@
 
 package org.apache.james.httpclient;
 
+import java.util.Optional;
+
 public class JwtToken {
 
-    private final String jwtTokenString;
+    private final Optional<String> jwtTokenString;
 
     public JwtToken(String jwtTokenString) {
-        this.jwtTokenString = jwtTokenString;
+        this.jwtTokenString = Optional.ofNullable(jwtTokenString);
```

Comment:
I dislike the fact that jwtTokenString could be null. Can we ask an optional in the constructor too?